.npmignore

```diff
@@ -9,4 +9,8 @@ database.json
 *.sublime-project
 *.sublime-workspace
 archive
-.db-migraterc
+coverage.html
+Makefile
+*.md
```

Comment:
I think md files have to stay, otherwise the npm info site will be empty.

Reply:
changelog however could be omitted, readme + license should be in the tar ball (also important for license scanners checking packages for the open source license in use)

Reply:
npm forces them to be, see the list of files in the initial post.

Reply:
Just read, http://npm.github.io/publishing-pkgs-docs/publishing/the-npmignore-file.html
Seems to be just fine